### ISSUE TITLE
[CoE Starter Kit - Upgrade Issue]

### ISSUE THREAD

**Issue by mathewam44** [microsoft/coe-starter-kit#8781]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

We are upgrading the COE starter kit from the core component version 3.18 which was released in 11/22/2021 package(sc attached). While tried to upgrade the core components to the latest version 4.32.2, we ended up with the below error. We have several questions regarding the upgrading activity.

1. Since we haven't updated the COE starter kit for more than 2 years, will there be any issues upgrading to the attached version 
2. Is it possible for us to delete all the existing solutions and Install the COE starter kit with latest solutions ?
3. While tried to update the starter kit, we received the attached error. Are we good to move forward with upgrading next solutions ? If not, please advise the instructions on how to resolve ?
4. Can we have a support ticket open to get assistance to upgrade this ?
![COE Existing in the Env](https://github.com/user-attachments/assets/dc952bce-c580-4efb-afbb-79ef8789d82d)

![COE LTS](https://github.com/user-attachments/assets/4ad14837-b3c5-4203-9e00-aa16bd29ff94)

![MicrosoftTeams-image (14)](https://github.com/user-attachments/assets/e00a63cc-8be6-42cc-88eb-4f300203a0af)

### What solution are you experiencing the issue with?

None

### What solution version are you using?

11/22/2021

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Unfortunately you will likely not be able to upgrade from that old of a build and will likely need to start from scratch in a clean environment, or as you say you could try to delete the solutions. A clean environment is always better if possible. 

Note that we do try to help with upgrade issues but just do not have the ability to assist with ugprades this old. 

Regarding support, the CoE Starter Kit is not a Microsoft product and so product support will not assist you. Note that there are partners which specialize in the CoE Starter Kit who can be hired, sometimes with ECIF funding if you qualify, to assist with the setup and configuration of the kit. You can find some via you can find them via [AppSource](https://appsource.microsoft.com/en/marketplace/consulting-services?product=powerapps&src=pa-m&search=coe&page=1)

In short, I recommend you start from scratch in a new solution at this point. Note we have a setup wizard now and so setting up is much less painful than a few years ago. 
[Before setting up the CoE Starter Kit](https://learn.microsoft.com/power-platform/guidance/coe/setup)

**Comment by Jenefer-Monroe** [comment]
Closing out as no further action for starter kit team
